docker/Dockerfile.tee-debug

```
@@ -0,0 +1,186 @@
+## CI/CD version
+## git clone, then cd into
+## docker image build -f docker/Dockerfile -t nexus.iex.ec/tee-worker-pre-compute:dev . --no-cache
+
+ARG HOME=/home
+ARG JAR_PATH=$HOME/build/libs/app-all.jar
+
+###
+### Build application jar
+###
+FROM gradle:6.8.3-jdk11 as gradle-builder
```

Why not building this image from the previous one?
```suggestion
ARG NATIVE_IMAGE
FROM ${NATIVE_MAGE}
```

---

Updated :heavy_check_mark: